specifyweb/frontend/js_src/lib/components/WorkBench/hooks.ts

```diff
@@ -187,7 +195,7 @@ export function getHotHooks(
           'UndoRedo.undo',
           'UndoRedo.redo',
         ].includes(source) ||
-        !workbench.hot ||
+        workbench.hot !== undefined ||
```

Comment:
```suggestion
        workbench.hot === undefined ||
```

We are canceling every `afterChange` hook event when `workbench.hot` is defined! 

Simple change to return to correct behavior 😃

Reply:
Woops my bad, thanks for catching that!